tests/integration/base.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import absolute_import
```

Comment:
I think it would be more consistent across the repo to use pytest form, rather than unittest form.

Reply:
This PR is more to address #546 than expand on the unit tests, which is to help set up guidance how others can set up integration tests. 

My preference is to keep the examples more generic than pytest specific, since not everything in python land uses pytest. With that said, I can change it to use pytest(i.e. consistent with everything else)...but that IMO implies that is the only supported way of testing.

Reply:
Yes, please use pytest